src/main/java/com/epam/test_generator/controllers/GeneralController.java

```diff
@@ -12,7 +12,7 @@
     @ApiOperation(value = "", hidden = true)
     @RequestMapping(value = "/")
     public String getMainPage() {
-        return "/newSuits";
+        return "index";
```

Comment:
Try to set project in a way this controller not required at all.

Reply:
General controller was deleted